build-package.sh

```diff
@@ -500,6 +504,7 @@ _show_usage() {
 	echo "  -o Specify directory where to put built packages. Default: output/."
 	echo "  --format Specify package output format (debian, pacman)."
 	echo "  --library Specify library of package (bionic, glibc)."
+	echo "   --disable-dependency-download-parallelizing disable dependency downloading parallelizing"
```

Comment:
extra space

Reply:
In your version of code it will run AFTER all packages are downloaded since you put it in the loop that is being invoked after all `wait -n` invocations.
Also the point of `--disable-dependency-download-parallelizing` is to fallback to old behaviour without explicitly add exit handler twice.

Reply:
I mean if parallelizing causes problems we should not use it at all.